packages/node/tests/common/alternatingFile.spec.ts

```diff
@@ -0,0 +1,96 @@
+import fs from 'fs';
+import path from 'path';
+import { AlternatingFileWriter } from '../../src/common/AlternatingFileWriter';
+
+function unlinkSafe(file: string) {
```

Comment:
One more idea for a test - when you dispose BacktraceClient, can you still add breadcrumbs? WIll it throw an error?

Reply:
This is out of scope of this test and review. I only implement the `BreadcrumbsStorage`, and not change anything about adding breadcrumbs.